efi/meson.build

```diff
@@ -159,36 +153,36 @@ else
   efi_format = ['--target=efi-app-@0@'.format(gnu_efi_arch)]
 endif
 
-libgcc_file_name = run_command(efi_cc, '-print-libgcc-file-name').stdout().strip()
+libgcc_file_name = run_command(cc.cmd_array(), '-print-libgcc-file-name').stdout().strip()
 efi_name = 'fwupd@0@.efi'.format(EFI_MACHINE_TYPE_NAME)
 
 o_file1 = custom_target('fwupdate.o',
                         input : 'fwupdate.c',
                         output : 'fwupdate.o',
-                        command : [efi_cc, '-c', '@INPUT@', '-o', '@OUTPUT@']
+                        command : [cc.cmd_array(), '-c', '@INPUT@', '-o', '@OUTPUT@']
                                   + compile_args)
 o_file2 = custom_target('fwup-debug.o',
                         input : 'fwup-debug.c',
                         output : 'fwup-debug.o',
-                        command : [efi_cc, '-c', '@INPUT@', '-o', '@OUTPUT@']
+                        command : [cc.cmd_array(), '-c', '@INPUT@', '-o', '@OUTPUT@']
                                   + compile_args)
 o_file3 = custom_target('fwup-efi.o',
                         input : 'fwup-efi.c',
                         output : 'fwup-efi.o',
-                        command : [efi_cc, '-c', '@INPUT@', '-o', '@OUTPUT@']
+                        command : [cc.cmd_array(), '-c', '@INPUT@', '-o', '@OUTPUT@']
                                   + compile_args)
 o_file4 = custom_target('fwup-common.o',
                         input : 'fwup-common.c',
                         output : 'fwup-common.o',
-                        command : [efi_cc, '-c', '@INPUT@', '-o', '@OUTPUT@']
+                        command : [cc.cmd_array(), '-c', '@INPUT@', '-o', '@OUTPUT@']
                                   + compile_args)
 
 o_file5 = custom_target('fwup-sbat.o',
                         output : 'fwup-sbat.o',
                         command : [
                           generate_sbat,
                           '@OUTPUT@',
-                          '--cc', efi_cc,
+                          '--cc', '@0@'.format(' '.join(cc.cmd_array())),
```

Comment:
do you need the @0@ or can you use `' '.join(cc.cmd_array())` here?

Reply:
fixed